.github/test-dispatch.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-BRANCH="sh-DR-1116-perf-action"
+BRANCH="sh-DR-1356-deploy-latest-to-perf"
```

Comment:
I'm assuming that this only ever gets used by PRs? I wonder if there's a way to pull this from the PR's branch if that's the case

Reply:
Good idea!